package.json

```diff
@@ -45,6 +45,7 @@
     "grunt": "^0.4.5",
     "grunt-shell": "^1.1.1",
     "gruntfile-gtx": "^0.3.0",
+    "lodash": "^3.10.1",
```

Comment:
I should note that there's no `lodash.lang` modular build. Any category level `lodash` stuff would require a custom build.